### PR TITLE
fix(app): update both thermocycler cutouts when removing for location conflict

### PR DESCRIPTION
**Pull request by brenthagen**
# Overview

when removing a thermocycler, to issue a valid deck configuration update we must update both cutouts A1 and B1. this substitutes a single left slot fixture in the paired cutout of the target location conflict cutout id. previously the `LocationConflictModal` was sending an update to remove the thermocycler module from only cutout A1 or cutout B1 and receiving a 422 response. This issue has likely been around since modules were added as fixtures, and updates some [special case logic that was intended to be altered](https://github.com/Opentrons/opentrons/pull/14425).

closes RQA-3074

## Test Plan and Hands on Testing

manually verified the fix

## Changelog

 - Updates both thermocycler cutouts when removing for location conflict

## Review requests

Add a thermocycler to deck configuration, then load a protocol with labware/other modules in A1 and/or B1. Fix the conflict in protocol setup.

## Risk assessment

low

### CODE REVIEW

File: app/src/atoms/SoftwareKeyboard/NumericalKeyboard/index.tsx [hunk 1]
```diff
@@ -37,7 +37,9 @@ export function NumericalKeyboard({
     <Keyboard
       keyboardRef={r => (keyboardRef.current = r)}
       theme={'hg-theme-default oddTheme1 numerical-keyboard'}
-      onInit={keyboard => {keyboard.setInput(initialValue)}}
+      onInit={keyboard => {
+        keyboard.setInput(initialValue)
+      }}
```
Comment (brenthagen) [inline app/src/atoms/SoftwareKeyboard/NumericalKeyboard/index.tsx]: to fix [chore_release-8.0.0](https://github.com/Opentrons/opentrons/tree/chore_release-8.0.0) formatting failing the lint-js check

Comment (vegano1) [reply app/src/atoms/SoftwareKeyboard/NumericalKeyboard/index.tsx]: Oops, my bad, though I merged the fix in.
Thank you!

File: app/src/organisms/Devices/ProtocolRun/SetupModuleAndDeck/LocationConflictModal.tsx [hunk 2]
```diff
@@ -111,14 +107,34 @@ export const LocationConflictModal = (
...
+            cutoutFixtureId: replacementCutoutFixtureId,
+            opentronsModuleSerialNumber: moduleSerialNumber,
+          }
+        } else if (
+          isThermocyclerCurrentFixture &&
+          ((cutoutId === 'cutoutA1' &&
+            existingCutoutConfig.cutoutId === 'cutoutB1') ||
+            (cutoutId === 'cutoutB1' &&
+              existingCutoutConfig.cutoutId === 'cutoutA1'))
+        ) {
```
Comment (CaseyBatten) [inline app/src/organisms/Devices/ProtocolRun/SetupModuleAndDeck/LocationConflictModal.tsx:127]: Of note, the Thermocycler Front and Rear fixtures belong to one FixtureGroup, identified in the ot3_standard.json deck definition file. In the future, if we have other multi-cutout fixtures, we may want to just remove all entities in a fixture group from the deck config and replace them with standard deck slots, as well as replacing the "selected cutout" with the new fixture the user is trying to add. Not needed for this PR, this solution definitely solves out current problem, but good to note for the future.

**Review by CaseyBatten — APPROVED** [review]
Looks good and just tested, works on Platform Expansion Flex. A comment below regarding our special casing.

**Pull request merged**